tests/conftest.py

```
@@ -11876,6 +11878,116 @@ def finalizer():
+        f"--region {region}",
+        use_yes=True,
+    )
+    bs_obj = BackingStore(
+        name=bs_name,
+        method="cli",
+        type="cloud",
+        uls_name=bucket_name,
+        mcg_obj=mcg_obj,
+    )
+    created["backingstore"] = bs_obj
+    log.info(f"Created backingstore {bs_name}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)